cmd/snap-repair/runner_test.go

```diff
@@ -693,7 +715,7 @@ scriptB
 
 AXNpZw==`}
 
-	mockServer := makeMockServer(c, &seqRepairs)
+	mockServer := makeMockServer(c, &seqRepairs, false)
```

Comment:
The downside of using naked true/false here is that one need to look at the signature of makeMockServer (same for TestNext) to see what the meaning of the bool is. Maybe we can use something like "const DoRedirectFirst = true"  and/or flags here to make it more obvious what semantic the bool values have in the arguments?